docs/QuickStartGuides/0-Overview.md

```diff
@@ -0,0 +1,4 @@
+# Overview
+
+The goal of Symbulation is to simplify the development of efficient,
```

Comment:
Change to "The goal of Symbulation is to provide an evolutionary agent-based framework for studying the evolution of symbiosis."